src/test/java/cloud/eppo/ConfigurationRequestorTest.java

```diff
@@ -58,7 +58,7 @@ public void testInitialConfigurationDoesntClobberFetch() throws IOException {
     String fetchedFlagConfig =
         FileUtils.readFileToString(differentFlagConfigFile, StandardCharsets.UTF_8);
 
-    when(mockHttpClient.getAsync("/api/flag-config/v1/config")).thenReturn(configFetchFuture);
+    when(mockHttpClient.getAsync("/flag-config/v1/config")).thenReturn(configFetchFuture);
```

Comment:
can this be Constants.FLAG_CONFIG_ENDPOINT or do you prefer the failsafe nature of the hardcoded string @aarsilv?